connections/connections_test.go

```diff
@@ -206,13 +206,62 @@ func Test174(t *testing.T) {
 	input := make(chan *sockets.SocketInfo, 0)
 	connections, servers := make(chan FullConnection, 100), make(chan ServerConnection, 100)
 
-	go New(false, true).filterAndPublish(false, 10*time.Second, input, connections, servers)
+	go New(false, true).handleSocketInfoChannel(false, 10*time.Second, input, connections, servers)
 
 	insert174data(t, input)
 
 	expectConnectionOnPort(t, connections, 35074)
 }
 
+func TestConnectionCleanupThatLastSeenMoreThan2AggregationIntervals(t *testing.T) {
```

Comment:
Name is a bit cryptic to me. I would rename it to something like `TestConnectionCleanupOnlyRemovesWhenOlderThan2AggregationIntervals`